.travis.yml

```diff
@@ -1,20 +1,35 @@
 # http://docs.travis-ci.com/user/languages/go/
 language: go
 
-go: 1.8
+go: 1.7
```

Comment:
I might experiment with go1.8 plugin, in which case this would _need_ to be `1.8` and not `1.7`. This work is not imminent, so this is a heads up.